src/lib/common/FileInput.svelte

```diff
@@ -19,30 +17,15 @@
   }
 </script>
 
-<!-- TODO Interactive elements inside a label are apparently invalid, but this works -->
-<label>
+<div class="govuk-form-group">
+  <label class="govuk-label" for="file-upload">{label}</label>
```

Comment:
Shouldn't we be using the form element defined below here?

Reply:
Ah good point, thanks! Fixed. I also brought back the unique ID, because `<label>` will break otherwise. We don't have any page with multiple file inputs, but we'll need to when we do